.github/workflows/docker.yaml

```diff
@@ -66,4 +66,4 @@ jobs:
         kubectl set image deployment.apps/celery-sms celery-sms=$DOCKER_SLUG:${GITHUB_SHA::7} -n=notification-canada-ca --kubeconfig=$HOME/.kube/config
     - uses: cds-snc/notification-pr-bot@master
       env:
-        TOKEN: ${{ secrets.TOKEN }}
+        TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
We should contact one of the organization's GitHub admin, to know what's the default permissions on GITHUB_TOKEN per repository and through the organization. If my understanding is correct, we need to know of the token's default permissions. It might be restricted to repository or through the whole organization. 

It might not be something desirable too from an admin perspective as we'd have permissions to write pull requests in all repositories under CDS, if my understanding is correct. But maybe we can make our case for PR bot usages? We are not the only team with one.

https://docs.github.com/en/actions/reference/authentication-in-a-workflow#permissions-for-the-github_token